### ISSUE TITLE
[pixels-core, pixels-presto] support 'is null' in query predicate.

### ISSUE THREAD

**Issue by bianhq** [pixelsdb/pixels#103]
Currently, using 'is null' in Presto may encounter the following error:
```java
com.facebook.presto.spi.PrestoException: Failed to prepare for read.
	at io.pixelsdb.pixels.presto.PixelsPageSource.getNextPage(PixelsPageSource.java:234)
	at com.facebook.presto.operator.ScanFilterAndProjectOperator.processPageSource(ScanFilterAndProjectOperator.java:273)
	at com.facebook.presto.operator.ScanFilterAndProjectOperator.getOutput(ScanFilterAndProjectOperator.java:234)
	at com.facebook.presto.operator.Driver.processInternal(Driver.java:379)
	at com.facebook.presto.operator.Driver.lambda$processFor$8(Driver.java:278)
	at com.facebook.presto.operator.Driver.tryWithLock(Driver.java:659)
	at com.facebook.presto.operator.Driver.processFor(Driver.java:272)
	at com.facebook.presto.execution.SqlTaskExecution$DriverSplitRunner.processFor(SqlTaskExecution.java:975)
	at com.facebook.presto.execution.executor.PrioritizedSplitRunner.process(PrioritizedSplitRunner.java:163)
	at com.facebook.presto.execution.executor.LegacyPrioritizedSplitRunner.process(LegacyPrioritizedSplitRunner.java:23)
	at com.facebook.presto.execution.executor.TaskExecutor$TaskRunner.run(TaskExecutor.java:492)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.IOException: Failed to prepare for read.
	at io.pixelsdb.pixels.core.reader.PixelsRecordReaderImpl.prepareBatch(PixelsRecordReaderImpl.java:601)
	at io.pixelsdb.pixels.presto.PixelsPageSource.getNextPage(PixelsPageSource.java:230)
	... 13 more
```

**Comment by bianhq** [comment]
If we use '=null' (although not meaningful) in query predicate, we encounter the following error:
```java
java.lang.NullPointerException: undefined
	at io.pixelsdb.pixels.core.reader.PixelsRecordReaderImpl.prepareRead(PixelsRecordReaderImpl.java:254)
	at io.pixelsdb.pixels.core.reader.PixelsRecordReaderImpl.prepareBatch(PixelsRecordReaderImpl.java:599)
	at io.pixelsdb.pixels.presto.PixelsPageSource.getNextPage(PixelsPageSource.java:230)
	at com.facebook.presto.operator.ScanFilterAndProjectOperator.processPageSource(ScanFilterAndProjectOperator.java:273)
	at com.facebook.presto.operator.ScanFilterAndProjectOperator.getOutput(ScanFilterAndProjectOperator.java:234)
	at com.facebook.presto.operator.Driver.processInternal(Driver.java:379)
	at com.facebook.presto.operator.Driver.lambda$processFor$8(Driver.java:278)
	at com.facebook.presto.operator.Driver.tryWithLock(Driver.java:659)
	at com.facebook.presto.operator.Driver.processFor(Driver.java:272)
	at com.facebook.presto.execution.SqlTaskExecution$DriverSplitRunner.processFor(SqlTaskExecution.java:975)
	at com.facebook.presto.execution.executor.PrioritizedSplitRunner.process(PrioritizedSplitRunner.java:163)
	at com.facebook.presto.execution.executor.LegacyPrioritizedSplitRunner.process(LegacyPrioritizedSplitRunner.java:23)
	at com.facebook.presto.execution.executor.TaskExecutor$TaskRunner.run(TaskExecutor.java:492)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by bianhq** [comment]
Anthoer related problem:
Query `select column_a from test_105 where column_b = null` will never stop.

**Comment by bianhq** [comment]
> Anthoer related problem:
> Query `select column_a from test_105 where column_b = null` will never stop.

We have this problem after fixing the first ('is null') problem. And we confirm that this is caused by the prepareBatch behavior and the Improper action taken by Presto. So we are going to partially resolve it in another issue.

**Comment by bianhq** [comment]
Resolved in PR #104.
